colors/nord.vim

```diff
@@ -591,6 +592,30 @@ call s:hi("CocErrorSign" , s:nord11_gui, "", s:nord11_term, "", "", "")
 call s:hi("CocInfoSign" , s:nord8_gui, "", s:nord8_term, "", "", "")
 call s:hi("CocHintSign" , s:nord10_gui, "", s:nord10_term, "", "", "")
 
+" vim-go
+" > fatih/vim-go
+call s:hi("goBuiltins", s:nord7_gui, "", s:nord7_term, "", "", "")
+hi! link goConstants Keyword
+hi! link goFunctionCall Function
+hi! link goFormatSpecifier FormatSpecifier
+hi! link goEscapeOctal FormatSpecifier
+hi! link goEscapeC FormatSpecifier
+hi! link goEscapeX FormatSpecifier
+hi! link goEscapeU FormatSpecifier
+hi! link goEscapeBigU FormatSpecifier
+hi! link goEscapeError FormatSpecifier
+" TODO: these are not exposed by vim-go so you need to add
```

Comment:
Please open an issue and/or a pull-request at https://github.com/fatih/vim-go for exposing those :)